setup.py

```diff
@@ -33,6 +33,7 @@
 
 tests_require = [
     'SQLAlchemy-Continuum>=1.2.1',
+    'SQLAlchemy-Utils~=0.0,>=0.33.0.',
```

Comment:
This is in ``test_require`` but I think it should be in ``install_requires`` instead (where the dependency is already). However, if you apply the above change then we don't need any changes to setup.py.